packages/interpolators/test/interpolatorLatency.spec.ts

```diff
@@ -68,17 +68,26 @@ describe("interpolator", () => {
       _isOnChangeCalledByUpdate: true,
     } as any
 
+    // we've just started and we're 50% between game and future game
+    // we're half way between 0 and 10, so should be a 5
     instance.update({ game: 0, futureGame: 10 })
     expect(instance.getPosition()).toEqual(5)
 
+    // we're still moving in the same direction and have accelerated to 0.5 (future speed = 1)
+    // we're half way between (0+0.5) and (10 + 1) = 5.75
     instance.update({ game: 10, futureGame: 12 })
-    expect(instance.getPosition()).toEqual(0.75)
+    expect(instance.getPosition()).toEqual(5.75)
 
+    // we're still moving in the same direction and have accelerated to 1 (future speed = 1.5)
+    // we're half way between (0.5+1) and (11 + 1.5) = 5.5 + 1.5 = 7
     instance.update({ game: 12, futureGame: 15 })
-    expect(instance.getPosition()).toEqual(2)
+    expect(instance.getPosition()).toEqual(7)
 
+    // we're still moving in the same direction and have accelerated to 1.5 (future speed = 2)
+    // we're half way between (1.5+1.5) and (12.5 + 2) - 12.5+2 is bigger than future game, so we'll use the future
+    // game state of 14. 50% between 3 and 14 = 8.5
```

Comment:
It seems like the actual position in logic goes from 15 to 14, i.e. the player moves in the opposite direction. Is that intentional?

If so, would be great to mention that in the comment. Since the test is already quite complex, it might be better to move the acceleration + move in opposite direction into another test that only focuses on that for just 2 time steps.

Reply:
It might be nice to just keep this test simple and do something like:
```
update({ game: 2, futureGame: 4 })
update({ game: 4, futureGame: 6 })
update({ game: 6, futureGame: 8 })
...
```

so this test focuses specifically on the interpolating using `timeSinceLastUpdate` and not anything else.

Reply:
It wasn't intentional. Fixed.